backend/api/Areas/Contacts/Mapping/Contact/AddressMap.cs

```diff
@@ -26,6 +26,25 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.Country, src => src.Address.Country.Description)
                 .Map(dest => dest.Postal, src => src.Address.PostalCode)
                 .Map(dest => dest.AddressType, src => src.AddressUsageTypeCodeNavigation);
+
+            config.NewConfig<Entity.PimsOrganizationAddress, Model.AddressModel>()
+                .Map(dest => dest.Id, src => src.AddressId)
+                .Map(dest => dest.RowVersion, src => src.ConcurrencyControlNumber)
+                .Map(dest => dest.StreetAddress1, src => src.Address.StreetAddress1)
+                .Map(dest => dest.StreetAddress2, src => src.Address.StreetAddress2)
+                .Map(dest => dest.StreetAddress3, src => src.Address.StreetAddress3)
+                .Map(dest => dest.Municipality, src => src.Address.MunicipalityName)
+                .Map(dest => dest.RegionId, src => src.Address.RegionCode)
+                .Map(dest => dest.Region, src => src.Address.RegionCodeNavigation.RegionName)
+                .Map(dest => dest.DistrictId, src => src.Address.DistrictCode)
+                .Map(dest => dest.District, src => src.Address.DistrictCodeNavigation.DistrictName)
+                .Map(dest => dest.ProvinceId, src => src.Address.ProvinceStateId)
+                .Map(dest => dest.Province, src => src.Address.ProvinceState.Description)
+                .Map(dest => dest.ProvinceCode, src => src.Address.ProvinceState.ProvinceStateCode)
```

Comment:
This is slightly risky business, as if any of these properties are null the mapping will NPE out. Are all of these fields required right now? if so this is technically fine, but isn't defensive.

Reply:
Ah good point. I am used to the compiler pointing to the possible nullability of the objects. I will add the checks.